storybookjs/storybook
# Perf: Reuse SVG icon paths by using symbols

## What I did

Instead of repeating the full SVG path for each icon in the sidebar, this defines an SVG `<symbol>` for each of the four icons, and then links to them with `<use>`.

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes, they shouldn't break.
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
